src/components/comments/CommentSection.js

```diff
@@ -1,26 +1,30 @@
 import React from "react";
 import Question from "./Question.js";
-import Answer from "./Answer.js";
 import "./comments.css";
 import "../mainContent/mainContent.css";
+
 export default function CommentSection(props) {
   /*The mapping function below takes each element within the followups prop array and will render it as 
 a corresponding question or answer*/
-  let allFollowups = props.followups.map((followup, index) => {
-    return followup.question ? (
-      <Question key={index} questionText={followup.question} />
-    ) : (
-      <Answer
-        key={index}
-        answerHeaderText={followup.answerHeader}
-        answerText={followup.answer}
+  let allFollowups = props.followups.filter((s, index) => index !== 0 && s.questionHeader).map((followup, index) => {
+    return (
+      <Question 
+        key={index} 
+        questionText={followup.question} 
+        questionHead={followup.questionHeader}
+        pic={props.followups[0]} /* passes profilePic to Question */
+        numAnswers={followup.numAnswers}
+
+        /* this is used to render answers to the question */
+        followups={props.followups}
       />
     );
   });
   /* Only rendering the ones that should be visible at that current moment*/
   let visibleFollowups = allFollowups.slice(0, props.currentVisibleText);
   return (
-    <div className="comments-container content-container " id={props.followups[0].question}>
+    <div className="comments-container content-container " id={props.followups[1].question}>
```

Comment:
The `id` attribute here serves as a unique "address" that allows the React Scroller to scroll to a specific comment section. `props.followups[0].question` used to be the string for the first question of each comment section, but I changed it to `props.followups[1].question` because the first question is now the second item in the array due to the profile picture being added as the first item (you can take a look at a `followups` section inside `labContent.js`).

Reply:
Oh gotcha, sounds good.

> because the first question is now the second item in the array due to the profile picture being added as the first item

Hm, that is a tad confusing - mentally, I prefer if arrays/lists are single-type. I'm not sure if using a dictionary makes things clearer on your end, but I'm envisioning something like keying the `props` object with `props.image`, and then `props.questions` - it should separate things a bit easier?